src/Voxel.MiddyNet/MiddyNetContext.cs

```diff
@@ -1,30 +1,36 @@
-﻿using System;
+﻿using Amazon.Lambda.Core;
+using System;
 using System.Collections.Generic;
-using Amazon.Lambda.Core;
 
 namespace Voxel.MiddyNet
 {
     public class MiddyNetContext
     {
         public ILambdaContext LambdaContext { get; private set; }
         public IDictionary<string, object> AdditionalContext { get; }
-        public List<Exception> MiddlewareExceptions { get; set; }
+        public List<Exception> MiddlewareBeforeExceptions { get; }
+        public List<Exception> MiddlewareAfterExceptions { get; }
+        public Exception HandlerException { get; set; }
         public IMiddyLogger Logger { get; private set; }
+        public Func<ILambdaLogger, IMiddyLogger> LoggerFactory { get; }
+
+        public bool HasExceptions => HandlerException != null || MiddlewareBeforeExceptions.Count > 0 || MiddlewareAfterExceptions.Count > 0;
```

Comment:
Count or Any?

Reply:
VS used to recomment using .Any(), but now it suggests using Count > 0. I'm happy with both.